<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchEngine</class>
 <widget class="QMainWindow" name="SearchEngine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchEngine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>281</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dropdown">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
